cpp/modmesh/io/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
`io` the name conflict with Python `io` module.  It is unnecessary ambiguity.  Please rename this directory as `inout`.

Reply:
I renamed the module name to `inout` in https://github.com/solvcon/modmesh/pull/233/commits/7432534167ee1da998cfb1a7024ed5e8f52f869f

Reply:
Thanks